package/MDAnalysis/coordinates/chemfiles.py

```diff
@@ -375,7 +391,17 @@ def _timestep_to_chemfiles(self, ts):
         if ts.has_velocities:
             frame.add_velocities()
             frame.velocities[:] = ts.velocities[:]
-        frame.cell = chemfiles.UnitCell(*ts.dimensions)
+
+        lengths = ts.dimensions[:3]
+        angles = ts.dimensions[3:]
+        if angles[0] == 0.0 and angles[1] == 0.0 and angles[2] == 0.0:
```

Comment:
@Luthaf I'm actually not sure what's being done here, do you have a quick TLDR you can provide?

Reply:
This might be me being overly defensive. Does MDA guarantee angles to have valid values? If so this check can probably be removed.

Reply:
Yes, chemfiles is fine with lengths of 0 but requires angles to be 90 in this case. I try to be generous in what code accept, so checking only for angles gives the maximal flexibility, but I'm also fine with checking if `ts.dimensions` is `[0., 0., 0., 0., 0., 0.]` and rejecting more wrong/missing cells.

Reply:
So what's the decision here? Should I change this to check for either `[0., 0., 0., 0., 0., 0.]` or `None`?